R/formula.R

```diff
@@ -0,0 +1,64 @@
+#' Building formula for model
+#'
+#' @description `r lifecycle::badge("experimental")`
+#'
+#' @param vars (`list`)\cr variables to use in the model
+#' @param cor_struct (`character`)\cr speicify the covariance structure to use
```

Comment:
```suggestion
#' @param cor_struct (`string`)\cr specify the covariance structure to use.
```
note: string here because the user should just specify a single covariance structure